app/components/billing-status-display/billing-status-badge.ts

```
@@ -0,0 +1,61 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import type AuthenticatorService from 'codecrafters-frontend/services/authenticator';
+import type FeatureFlagsService from 'codecrafters-frontend/services/feature-flags';
+import type RouterService from '@ember/routing/router-service';
+import type UserModel from 'codecrafters-frontend/models/user';
+import type PromotionalDiscountModel from 'codecrafters-frontend/models/promotional-discount';
+
+interface Signature {
+  Element: HTMLDivElement;
+  Args: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)